src/Symfony/Component/Form/Resources/translations/validators.tr.xlf

```diff
@@ -14,6 +14,126 @@
+                <source>The hidden field is invalid.</source>
+                <target>Gizli alan geçersiz.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Lütfen bir tam sayı girin.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Lütfen geçerli bir dil seçin.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Lütfen geçerli bir yerel ayar seçin.</target>
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please enter a valid money amount.</source>
+                <target>Lütfen geçerli bir para tutarı girin.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>Please enter a number.</source>
+                <target>Lütfen bir numara giriniz.</target>
+            </trans-unit>
+            <trans-unit id="116">
+                <source>The password is invalid.</source>
+                <target>Şifre geçersiz.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>Please enter a percentage value.</source>
+                <target>Lütfen bir yüzde değeri girin.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>The values do not match.</source>
+                <target>Değerler eşleşmiyor.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>Please enter a valid time.</source>
+                <target>Lütfen geçerli bir zaman girin.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>Lütfen geçerli bir saat dilimi seçin.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>Lütfen geçerli bir giriniz URL.</target>
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid search term.</source>
+                <target>Lütfen geçerli bir arama terimi girin.</target>
+            </trans-unit>
+            <trans-unit id="123">
+                <source>Please provide a valid phone number.</source>
+                <target>lütfen geçerli bir telefon numarası sağlayın.</target>
+            </trans-unit>
+            <trans-unit id="124">
+                <source>The checkbox has an invalid value.</source>
+                <target>Onay kutusunda geçersiz bir değer var.</target>
+            </trans-unit>
+            <trans-unit id="125">
+                <source>Please enter a valid email address.</source>
+                <target>Geçerli bir e.</target>
```

Comment:
Can you double check this?

Reply:
Done! 👍 I have just rectified it, now it's much better understandable:
please =lütfen 
Enter = girin
Valid  = geçerli
email address = e-posta adresi

Reply:
Lovely!